src/python_routing_v01/compute_nhd_routing_SingleSeg.py

```diff
@@ -254,7 +254,7 @@ def in_wsl() -> bool:
     WSL is thought to be the only common Linux kernel with Microsoft in the name.
     """
 
-    return "microsoft" in (os.uname().release).lower()
```

Comment:
need to leave this line with the `.lower()`

Reply:
i pushed two commits that change these, but I can't see it updated in files changed. Only in the commits tab.